src/app/components/MostRead/Label/index.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render } from '../../react-testing-library-with-providers';
+import { ServiceContextProvider } from '../../../contexts/ServiceContext';
+import MostReadSectionLabel from '.';
+
+describe('MostReadSectionLabel assertion', () => {
+  it('should render most-read section label with correct attributes', async () => {
+    const { container } = render(
+      <ServiceContextProvider service="persian">
+        <MostReadSectionLabel />
+      </ServiceContextProvider>,
+    );
+    const label = container.getElementsByTagName('span')[2];
```

Comment:
Not sure the scope of this migration, but would be nice to see these built in JS methods swapped out for testing library queries https://testing-library.com/docs/queries/about/#types-of-queries. But if it's not in the scope no need to worry about it

Reply:
Yep, agree. A lot of our newer tests have been written this way and I recall agreeing with JR to avoid using JS methods unless necessary. Whilst it doesn't mention it explicitly in our standards (https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/Testing.stories.mdx), the examples in there use library queries instead of JS methods.